pkg/kubectl/cmd/attach_test.go

```diff
@@ -228,7 +228,6 @@ func TestAttach(t *testing.T) {
 					body := objBody(codec, test.pod)
 					return &http.Response{StatusCode: 200, Header: defaultHeader(), Body: body}, nil
 				default:
-					// Ensures no GET is performed when deleting by name
```

Comment:
This comment is not right here.
It says `Ensures no GET is performed`, but `GET` does performed in line 224